devicetypes/smartthings/springs-window-fashions-shade.src/springs-window-fashions-shade.groovy

```diff
@@ -266,4 +261,29 @@ def refresh() {
             zwave.switchMultilevelV1.switchMultilevelGet().format(),
             zwave.batteryV1.batteryGet().format()
     ], 1500)
+}
+
+def levelChangeFollowUp(expectedLevel) {
+    state.expectedValue = expectedLevel
+    state.levelChecks = 0
+    runIn(5, "checkLevel", [overwrite: true])
+}
+
+def checkLevelReport(value) {
+    if (state.expectedValue != null) {
+        if ((state.expectedValue == 99 && value >= 99) ||
+                (state.expectedValue >= value - 2 && state.expectedValue <= value + 2 )) {
```

Comment:
Looks like ya got an extra space before the )) { (and below). Otherwise it looks good! I have the test program running this change. I'll copy over all of the code to make sure that I didn't leave anything out and let it run.